docs/semgrep-ci/overview.md

```diff
@@ -145,13 +145,13 @@ To set up notifications, see [Notifications document](/semgrep-app/notifications
 Notifications are sent only the first time a given finding is seen. [See how notifications are de-duplicated](/semgrep-app/notifications/#de-duplication)
 :::
 
-#### Security Dashboards
```

Comment:
This section header was plural because if you look below, we list three possible dashboards one could use

> - [GitLab SAST Security Dashboard](https://docs.gitlab.com/ee/user/application_security/security_dashboard/) (requires GitLab Ultimate subscription)
> - [GitHub Advanced Security Dashboard](https://docs.github.com/en/github/getting-started-with-github/learning-about-github/about-github-advanced-security) (requires GitHub Enterprise subscription)
> - [Semgrep App](https://semgrep.dev/manage) (free and paid tiers available)

Reply:
@underyx thank you for letting me know, edited accordingly.